presto-docs/src/main/sphinx/develop/presto-authenticator.rst

```diff
@@ -0,0 +1,52 @@
+===========================
+Custom Presto Authenticator
+===========================
+
+Presto supports authentication via a custom presto authenticator
```

Comment:
```suggestion
Presto supports authentication through a custom Presto authenticator
```
Don't use `via`. See the GitLab documentation style guide word list entry for via for discussion and alternate working suggestions, if the suggested `through` isn't a good fit for you.